### ISSUE TITLE
Uncaught TypeError: Cannot read property 'common' of undefined

### ISSUE THREAD

**Issue by Dases95** [KillerCodeMonkey/ngx-quill#527]
I 'm using 
<quill-editor style="height: 250px" formControlName ='editor' class="formGroup">
to resize the editor  , the editor won't resize and this error appear
 Uncaught TypeError: Cannot read property 'common' of undefined :ngx-quill.umd.js:4
    at ngx-quill.umd.js:4
    at ngx-quill.umd.js:5

**Comment by Dases95** [comment]
im using 
"<quill-editor style="height: 250px" formControlName ='editor' class="formGroup">"

**Comment by Dases95** [comment]
quill-editor style="height: 250px" formControlName ='editor' class="formGroup"

**Comment by KillerCodeMonkey** [comment]
first there should not be white spaces between `formControlName="editor"`

and keep in mind to install peer dependencies of ngx-quill mentioned in the readme (angular core, common quilljs)

**Comment by Dases95** [comment]
i have installed all the dependencies but the problem still

**Comment by KillerCodeMonkey** [comment]
which ngx-quill version do you use. And did you checked the demo repo?

There you have a full example how to use ngx-quill

**Comment by Dases95** [comment]
+ quill@1.3.6
+ ngx-quill@7.3.1

**Comment by KillerCodeMonkey** [comment]
do you have an example repo for me or a stackblitz?

**Comment by Dases95** [comment]
**`app.module`**
QuillModule.forRoot({
      modules: {
        syntax: false,
        toolbar: [
          ['bold', 'italic', 'underline', 'strike'] ,
          [ 'code-block'],       
          [{ 'header': 1 }, { 'header': 2 }],   
          [{ 'list': 'ordered'}, { 'list': 'bullet' }],
          [{ 'script': 'sub'}, { 'script': 'super' }],
          [{ 'indent': '-1'}, { 'indent': '+1' }],          // outdent/indent
          [{ 'direction': 'rtl' }],
                                             // remove formatting button
      
          ['link', 'image', 'video']  
 
      ]
      }
    }),
**HTML**
<form class="col s12" [formGroup]="editorForm" (ngSubmit)="onSubmit()">
            <quill-editor style="height: 250px" formControlName ="editor" class="formGroup">
            
            </quill-editor>
         
            <br>
            <button class="btn waves-effect waves-light right teal darken-2" type="submit" name="action">Ajouter
                    <i class="material-icons left ">person_add</i>
                  </button>
        </form>
**component**
`import { Component, OnInit } from '@angular/core';
import { FormGroup, FormControl } from '@angular/forms';
declare var M:any;
@Component({
  selector: 'app-add-fiche-information',
  templateUrl: './add-fiche-information.component.html',
  styleUrls: ['./add-fiche-information.component.css']
})
export class AddFicheInformationComponent implements OnInit {
  editorForm : FormGroup
 
  constructor() { }

  ngOnInit() {
    this.editorForm = new FormGroup({
       'editor': new FormControl(null)
    });
   
  }

}
`

**Comment by KillerCodeMonkey** [comment]
and do you import FormModule in your app.module.ts

**Comment by Dases95** [comment]
yes

**Comment by Dases95** [comment]
i' dont undestand where to put this 
`set format and dependencies in packages:
packages: {
  'ngx-quill': {
    format: 'cjs',
    meta: {
      deps: ['quill']
    }
  },
  'quill': {
    format: 'cjs'
  }
}`

**Comment by KillerCodeMonkey** [comment]
are you using systemjs? in general when using angular cli you do not have to set this.

**Comment by Dases95** [comment]
non i'm using angular cli

**Comment by KillerCodeMonkey** [comment]
yeah then you do not need this configuration ;-)

**Comment by Dases95** [comment]
<img width="959" alt="Capture2" src="https://user-images.githubusercontent.com/26898071/65372754-485ecc00-dc74-11e9-9dca-f8555f5e2c68.PNG">

**Comment by KillerCodeMonkey** [comment]
yeah like i said angular cli should not load the umd-file. so you have misconfigured your build.

just follow https://github.com/KillerCodeMonkey/ngx-quill#for-standard-webpack-angular-cli-and-tsc-builds

and thats it. you can see this in the demo repo lined in the readme.

**Comment by KillerCodeMonkey** [comment]
any updates?

**Comment by KillerCodeMonkey** [comment]
closed to inactivity

**Comment by riyagarg47** [comment]
We just need "node_modules/quill/dist/quill.js" in scripts array of angular.json for angular cli >=6  as mentioned at https://github.com/KillerCodeMonkey/ngx-quill#for-standard-webpack-angular-cli-and-tsc-builds . There is no need of umd.js . This worked for me. Thanks @KillerCodeMonkey
